.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+name: build the package
+on:
+  pull_request:
+jobs:
+  run:
+    name: build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Go build
```

Comment:
I think we should have a `run: go version` step after setting up Go, since we have a floating version.

Reply:
yes added new step